impl/src/main/java/org/jboss/arquillian/warp/impl/client/execution/DefaultWarpRequestSpecifier.java

```diff
@@ -69,12 +69,11 @@ public WarpExecutionBuilder initiate(Activity activity) {
      * (non-Javadoc)
      * @see org.jboss.arquillian.warp.client.execution.SingleVerificationSpecifier#verify(org.jboss.arquillian.warp.ServerInspection)
      */
-    @SuppressWarnings("unchecked")
     public <T extends Inspection> T inspect(T inspection) {
         initializeSingleGroup();
         singleGroup.addInspections(inspection);
         WarpResult result = execute();
-        return (T) result.getGroup(SingleInspectionSpecifier.GROUP_ID).getInspection();
+        return result.getGroup(SingleInspectionSpecifier.GROUP_ID).<T>getInspection();
```

Comment:
<T> in front of getInspection() can be omitted too.

Reply:
Done. Generics in Java are strange ;-) - I am more used to the implementation in .NET.

Reply:
I think the recent failed test is a random one and not related to my latest change. Shouldn't there be an option to re-run the tests? Seems I don't have enough privileges.